joomla/joomla-cms
# [4.0] Remove show_associations field of association is not enabled

Pull Request for Issue # .

### Summary of Changes
This simple PR removes the field show_associations field of association is not enabled. It causes fatal error while saving article when I tested PR #12414 and @Hackwar had to add a check (which I think not necessary) to prevent the error.

Plus, it also removes some unused variables/lines of code which can be seen easily if you use a good IDE like phpstorm.

### Testing Instructions
Apply patch, try to save article and confirm nothing broken. Maybe code review should be enough.
